migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -56,6 +59,26 @@
       description = "The BCP-47 country code the given phone number(s) belong to (e.g. 44)")
   String countryCode;
 
+  @ArgGroup()
+  OptionalParameterType optionalParameter;
+
+  static class OptionalParameterType {
+    @Option(names = {"-l", "--lenientExport"},
```

Comment:
How about being more explicit with the parameter name so it's a bit more self-explanatory, e. g. `--allowInvalidPhoneNumbers`?